### PR TITLE
Resolve linker error in transaction_processor test

### PR DESCRIPTION
**Pull request by motxx**
## What is this pull request?
Resolve linker error of transaction processor test

## Why do you implement it? Why do we need this pull request?
To pass test.
  
## Details/Features
reference:
https://stackoverflow.com/questions/3065154/undefined-reference-to-vtable

## P.S.
Other tests will be fixed in other PRs.

### CODE REVIEW

File: irohad/torii/processor/transaction_processor.hpp [hunk 1]
```diff
@@ -33,11 +33,13 @@ namespace iroha {
     class TransactionProcessor {
       public:
 
+      virtual ~TransactionProcessor() {}
+
```
Comment (muratovv) [inline irohad/torii/processor/transaction_processor.hpp]: `virtual ~TransactionProcessor() = default;` better

File: irohad/torii/processor/transaction_processor.hpp [hunk 2]
```diff
@@ -33,11 +33,13 @@ namespace iroha {
...
       public:
 
+      virtual ~TransactionProcessor() {}
+      
       /**
        * Add transaction to the system for processing
        * @param transaction - transaction for processing
        */
-      virtual void handle(model::Client& client,model::Transaction &transaction);
+      virtual void handle(model::Client& client,model::Transaction &transaction) {}
```
Comment (muratovv) [inline irohad/torii/processor/transaction_processor.hpp]: May be write handle as pure abstract function?
`virtual void handle(model::Client& client,model::Transaction &transaction) = 0;`

Comment (motxx) [inline irohad/torii/processor/transaction_processor.hpp]: Ah, just now I fixed.

**Comment by motxx** [comment]
Already merged. I'll close PR.

**Pull request closed without merging**